beetmoverscript/templates/firefox_nightly_en_us_unsigned.yml

```diff
@@ -4,70 +4,50 @@ mapping:
   # common deliverables
 {% for locale in ['en-US'] %}
   {{ locale }}:
-    common:
-      artifact: target.common.tests.zip
+    target.common.tests.zip:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.common.tests.zip
-    cppunittest:
-      artifact: target.cppunittest.tests.zip
+    target.cppunittest.tests.zip:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.cppunittest.tests.zip
-    crashreporter:
-      artifact: target.crashreporter-symbols.zip
+    target.crashreporter-symbols.zip:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.crashreporter-symbols.zip
-    json:
-      artifact: target.json
+    target.json:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.json
-    mochitest:
-      artifact: target.mochitest.tests.zip
+    target.mochitest.tests.zip:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.mochitest.tests.zip
-    mozinfo:
-      artifact: target.mozinfo.json
+    target.mozinfo.json:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.mozinfo.json
-    reftest:
-      artifact: target.reftest.tests.zip
+    target.reftest.tests.zip:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.reftest.tests.zip
-    talos:
-      artifact: target.talos.tests.zip
+    target.talos.tests.zip:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.talos.tests.zip
-    test_packages:
-      artifact: target.test_packages.json
+    target.test_packages.json:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.test_packages.json
-    txt:
-      artifact: target.txt
+    target.txt:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.txt
-    webplatform:
-      artifact: target.web-platform.tests.zip
+    target.web-platform.tests.zip:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.web-platform.tests.zip
-    xpcshell:
-      artifact: target.xpcshell.tests.zip
+    target.xpcshell.tests.zip:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.xpcshell.tests.zip
-    info:
-      artifact: target_info.txt
+    target_info.txt:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}_info.txt
-    jsshell:
-      artifact: target.jsshell.zip
+    target.jsshell.zip:
       s3_key: jsshell-{{ platform }}.zip
-    mozharness_package:
-      artifact: mozharness.zip
+    mozharness.zip:
       s3_key: mozharness.zip
-    xpi:
-      artifact: {{ platform }}/xpi/firefox-{{ version }}.{{ locale }}.langpack.xpi
-      s3_key: firefox-{{ version }}.{{ locale }}.langpack.xpi
-    mar_tools_mar:
-      artifact: host/bin/mar
+    # uncomment when `Bug 1329310 - Support SIMPLE NAME for langpacks too` is resolved
+    #    target.langpack.xpi:
+    #      s3_key: firefox-{{ version }}.{{ locale }}.langpack.xpi
+    mar:
```

Comment:
yes, while it seems like this code would mean the mar/mbsdiff artifacts would live in `work_dir/cot/{task_id}/{mar,mbsdiff}`, they actually don't. They are still at `work_dir/cot/{task_id}//host/bin/{mar,mbsdiff}` or whatever. This is because 'mar' and 'mbsdiff' is just the key in context.artifacts_to_beetmove[locale] dict that maps artifact basename to full abs_paths.

see:

- https://github.com/mozilla-releng/beetmoverscript/blob/cot/beetmoverscript/task.py#L48
- https://github.com/mozilla-releng/beetmoverscript/blob/cot/beetmoverscript/script.py#L58-L60

Does this clear things up or did I miss something in my logic?